light/client.go

```diff
@@ -755,6 +781,15 @@ func (c *Client) bisectionAgainstPrimary(
 
 	switch errors.Unwrap(err).(type) {
 	case ErrInvalidHeader:
+		// If the target header is invalid, return immediately.
+		invalidHeaderHeight := err.(ErrVerificationFailed).To
```

Comment:
Isn't the err type `ErrInvalidHeader`? why can't you just use `err.To == newHeader.Height` like you did for sequential verification

Reply:
no, it's `error` interface